extensions/search-astro-docs/src/index.tsx

```diff
@@ -0,0 +1,59 @@
+import { Action, ActionPanel, Icon, List, openExtensionPreferences } from "@raycast/api";
+import { useState } from "react";
+import { URL } from "url";
+import { documentationListV3 as docsList } from "./data/docs";
+
+export default function UserSearchRoot() {
+  const [search, setSearch] = useState<string>();
+  return (
+    <List searchBarPlaceholder="Search the Astro Documentation" onSearchTextChange={setSearch} filtering={true}>
+      <List.EmptyView
+        title="No Results"
+        icon="noview.png"
+        description={`Search '${search}' in the Astro Documentation`}
+        actions={
+          <ActionPanel>
+            <OpenSearchInBrowserAction search={search ?? ""} />
+          </ActionPanel>
+        }
+      />
+      {docsList.map((docsItem) => (
+        <List.Item
+          keywords={docsItem.keywords}
+          key={docsItem.title}
+          title={`${docsItem.title}`}
+          icon={docsItem.icon ?? "astro-search-icon.png"}
+          actions={
+            <ActionPanel>
+              <Action.OpenInBrowser
+                title="Open the Astro Documentation"
+                url={docsItem.url ? docsItem.url : `http://a.stro.cc/${docsItem.title}`}
+              />
+            </ActionPanel>
+          }
+        />
+      ))}
+      <List.Item
+        key={"preferences"}
+        keywords={["settings", "language"]}
+        icon={Icon.Gear}
+        title={"Open extension preferences"}
+        actions={
+          <ActionPanel>
+            <Action
+              title="Open Extension Preferences"
+              onAction={openExtensionPreferences}
+              icon={Icon.Gear}
+              shortcut={{ modifiers: ["cmd", "shift"], key: "," }}
+            />
+          </ActionPanel>
+        }
+      />
```

Comment:
There is no other place in Raycast that adds it as a List.Item, I would add it as an action on the looped items instead 🙂

Reply:
When I add it in the loop, it doesn't show up for me when I do a search for it

Reply:
It will only be available in the action panel, do you like having it in the list instead?

Reply:
Yes! I would like it to be in the list instead. I think it's more accessible / easier that way